package.json

```diff
@@ -33,7 +33,18 @@
       "transform-object-rest-spread"
     ],
     "presets": [
-      "es2015"
+      [
+        "env",
+        {
+          "targets": {
+            "browsers": [
+              "last 3 Chrome versions",
+              "last 3 Firefox versions",
+              "last 3 Opera versions"
```

Comment:
3 versions sounds good to me. If someone complains, we can always change it later.

Yeah, I think specifying only the current build target is overkill. If the goal is to reduce bundle size, there's easier things to do like minifying the dist/ .js files and/or removing sourcemaps from the out/ .zip files.